### PR TITLE
Orm graph

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: solar/solar/interfaces/db/base.py [hunk 1]
```diff
@@ -127,12 +127,12 @@ class BaseGraphDB(object):
...
     COLLECTIONS = Enum(
         'Collections',
-        'input resource state_data state_log plan_node plan_graph events stage_log commit_log'
+        'input resource state_data state_log plan_node plan_graph plan_edge events stage_log commit_log'
     )
     DEFAULT_COLLECTION=COLLECTIONS.resource
     RELATION_TYPES = Enum(
         'RelationTypes',
-        'input_to_input resource_input plan_edge graph_to_node resource_event'
+        'input_to_input resource_input plan_to_node plan_to_edge resource_event'
```
Comment (loles) [inline solar/solar/interfaces/db/base.py]: We need to document this. Not necessarily in this patch

Comment (pigmej) [inline solar/solar/interfaces/db/base.py]: +1

File: solar/solar/test/test_graph_api.py [hunk 2]
```diff
@@ -34,7 +35,7 @@ def test_simple_plan_created_and_loaded(simple):
     plan = graph.get_plan(simple.graph['uid'])
     assert set(plan.nodes()) == {'just_fail', 'echo_stuff'}
 
-
+@mark.xfail
```
Comment (pigmej) [inline solar/solar/test/test_graph_api.py]: Why ?

**Comment by loles** [comment]
looks good to me

**Comment by pigmej** [comment]
It works but it's terrible slow for now. We need to change backend to speedup it.

**Pull request closed without merging**